beacon-chain/powchain/log_processing.go

```diff
@@ -34,6 +34,13 @@ const eth1DataSavingInterval = 100
 const maxTolerableDifference = 50
 const defaultEth1HeaderReqLimit = uint64(1000)
 const depositlogRequestLimit = 10000
+const batchSizeAdditiveFactorMultiplier = 0.10
+const batchSizeMultiplicativeDecreaseDivisor = 2
```

Comment:
@nisdas please feel free to provide good names for these 2 constants

Reply:
I think removing the `batchSize` prefix for both variables should be fine.
```
const additiveFactorMultiplier = 0.10
const multiplicativeDecreaseDivisor = 2
```